README.md

```diff
@@ -64,12 +65,17 @@ Ensure your shell has access to the tools listed above, see [Get started on macO
   $ idf.py set-target esp32c6
   ```
 
-3. Build and deploy the application to your device. 
+or (if using ESP32C3)
```

Comment:
instead of a new "```shell" block can we note this in the text "Configure the build system for your microcontroller"?

I'd also like to see this reflected the the tutorial content as well